client-js/src/client/backend-client.js

```diff
@@ -176,6 +205,18 @@ export class BackendClient {
 
     return new FirebaseBackendClient(endpoint, firebaseClient, requestFactory)
   }
+
+  /**
+   * Creates a new Fetch object specifying the target of fetch and its parameters.
+   *
+   * @param {!TypedMessage<Query>} query a query processed by Spine
+   * @returns BackendClient.Fetch<T> an object that performs the fetch
+   * @template <T> type of Fetch results
+   * @private
```

Comment:
- Seems it is should be more correct to declare it as `@protected` since it is meant to be overridden in subclasses.
- We also usually add [`@abstract`](http://usejsdoc.org/tags-abstract.html) in such cases (to make the method's status more clear).